happi/device.py

```diff
@@ -382,3 +356,44 @@ def __repr__(self):
 
     def __eq__(self, other):
         return (self.prefix, self.name) == (other.prefix, other.name)
+
+
+class Device(HappiItem):
```

Comment:
We should pick a reasonable rename alias for this when we transition away from using this name